assets/base/i18n/en.json

```diff
@@ -333,6 +333,8 @@
   "mobile.oauth.switch_to_browser.title": "Redirecting...",
   "mobile.oauth.try_again": "Try again",
   "mobile.open_gm.error": "We couldn't open a group message with those users. Please check your connection and try again.",
+  "mobile.participants.header": "THREAD PARTICIPANTS",
+  "mobile.participants.you": "(you)",
```

Comment:
are we repeating this string? it already exists for `suggestion.mention.you` and I really don't see this one changing to a different value any time soon and the one I'm talking about already has a space in it, so no need to go crazy.

if you want I'm up for changing the key of the string for a more generic name but 0/5.

In fact as you'll need to use  the `user_item` component, this is already implemented there and you won't need it anyway.